samples/esb/esb_ptx/src/main.c

```diff
@@ -97,7 +98,44 @@ int clocks_start(void)
 	LOG_DBG("HF clock started");
 	return 0;
 }
-#endif /* defined(CONFIG_CLOCK_CONTROL_NRF) */
+
+#elif defined(CONFIG_CLOCK_CONTROL_NRF2)
+
+int clocks_start(void)
+{
+	int err;
+	int res;
+	const struct device *radio_clk_dev =
+		DEVICE_DT_GET_OR_NULL(DT_CLOCKS_CTLR(DT_NODELABEL(radio)));
+	struct onoff_client radio_cli;
+
+	/** Keep radio domain powered all the time to reduce latency. */
+	nrf_lrcconf_poweron_force_set(NRF_LRCCONF010, NRF_LRCCONF_POWER_DOMAIN_1, true);
+
+	sys_notify_init_spinwait(&radio_cli.notify);
+
+	err = nrf_clock_control_request(radio_clk_dev, NULL, &radio_cli);
+
+	do {
+		err = sys_notify_fetch_result(&radio_cli.notify, &res);
+		if (!err && res) {
+			LOG_ERR("Clock could not be started: %d", res);
+			return res;
+		}
+	} while (err == -EAGAIN);
+
+#if defined(NRF54L15_XXAA)
```

Comment:
This section is repeated from the other branch of `#if defined(CONFIG_CLOCK_CONTROL_NRF)`. How is the selection between `CONFIG_CLOCK_CONTROL_NRF` and `CONFIG_CLOCK_CONTROL_NRF2` determined for nRF54L15? Can it use both clock control implementations, or can we avoid  the duplicate?

Reply:
For CONFIG_CLOCK_CONTROL_NRF2 it is not needed.